README.md

```diff
@@ -550,38 +591,100 @@ That version is available in the context object passed to your configuration mod
 Here, for example, is how you could use the `DefinePlugin` in your `underreact.config.js`:
 
 ```js
+// underreact.config.js
 module.exports = ({ webpack }) => {
   return {
     webpackPlugins: [new webpack.DefinePlugin(..)]
   };
 }
 ```
 
-### webpackConfigTransform
-
-Type: `config => transformedConfig`. Default `x => x` (identify function).
+## FAQs
 
-If you want to make changes to the Webpack configuration beyond what's available in the above options, you can use this, the nuclear option.
-Your function receives the Webpack configuration that Underreact generates and returns a new Webpack configuration, representing your heart's desires.
+### How do I make Jest use Underreact's Babel configuration ?
 
-**Tip:** You should think twice before using `webpackConfigTransform`, as Underreact tries its best to abstract away Webpack so that you can focus on the important part of building your application.
+Underreact requires Jest version **>=23.6** and the steps mentioned for using Jest with **Babel 7** in the [official installation docs](https://jestjs.io/docs/en/getting-started#using-babel). Once done, you can follow the steps in [Exposing babel.config.js](#exposing-babelconfigjs).
```

Comment:
So is it required that you expose `babel.config.js` if you want to use Jest? If so we should be more direct about that.

Reply:
Yes Jest or any tool depending on Babel would need config exposed. I can make it more explicit.